README.md

```diff
@@ -113,17 +116,22 @@ In the example above, two factor authentication will not be required for local
 users.
 
 This gem is compatible with [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en).
-You can generate provisioning uris by invoking the following method on your model:
+To enable this a shared secret must be generated by invoking the following
+method on your model:
+
+```ruby
+user.create_otp_secret
+```
+
+This can then be shared via a provisioning uri:
 
 ```ruby
 user.provisioning_uri # This assumes a user model with an email attribute
 ```
 
 This provisioning uri can then be turned in to a QR code if desired so that
 users may add the app to Google Authenticator easily.  Once this is done, they
-may retrieve a one-time password directly from the Google Authenticator app as
-well as through whatever method you define in
-`send_two_factor_authentication_code`.
+may retrieve a one-time password directly from the Google Authenticator app.
 
 #### Overriding the view
 
```

Comment:
We should also update the `Automatic initial setup` and `Manual initial setup` sections to mention the new DB columns.